src/main/webapp/dev/js/common/remindModal.js

```diff
@@ -2,6 +2,19 @@ import {
     sendRemindersToStudents,
 } from './instructor';
 
+function bindCheckAllCheckboxesEventHandler() {
+    const $checkAllSubmitted = $('#remind-particular-checkall-submitted');
+    const $checkAllNotSubmitted = $('#remind-particular-checkall-notsubmitted');
+    $checkAllSubmitted.on('click', () => {
+        const $studentsResponded = $('#studentList').find('.bg-info').find('input[type="checkbox"]');
```

Comment:
Why using `bg-info` (and later on `bg-danger`) as selectors, though? Those are generic Bootstrap classes and does not look safe.

Reply:
Agree. I think these should be replaced with classes instead.
I was trying to follow the code below:
https://github.com/bqnguyen94/teammates/blob/c49412c0ff0eef1a8b33323c6e2df917eca8cdb4/src/main/webapp/dev/js/common/remindModal.js#L18-L29
Should I add classes and apply to this code portion as well?

Reply:
Hmm no idea why the code snippet doesn't appear...

Reply:
@bqnguyen94 feel free to, since it is in reasonable proximity.

Reply:
I agree that the `remindModal` class is good to have, but since you are changing the selectors in `bindCheckAllCheckboxesEventHandler`, it is better to change the `.bg-danger` in the `remindModal` class as well